game.js

```diff
@@ -0,0 +1,138 @@
+let helpers = require("./helpers.js");
+let lastLetter = "";
+let cities = [
+  "Архангельск",
+  "Борисполь",
+  "Воронеж",
+  "Гюмри",
+  "Донецк",
+  "Енакиево",
+  "Жуковский",
+  "Запорожье",
+  "Изюм",
+  "Йошкар-Ола",
+  "Киев",
+  "Лондон",
+  "Москва",
+  "Николаев",
+  "Орёл",
+  "Париж",
+  "Ростов",
+  "Селидово",
+  "Трускавец",
+  "Уфа",
+  "Флоренция",
+  "Харьков",
+  "Цюрих",
+  "Челябинск",
+  "Шанхай",
+  "Щёлково",
+  "Ыгдыр",
+  "Электросталь",
+  "Юрмала",
+  "Ялта"
+].map(item => item.toLowerCase());
+
+let sessions = {
+  makeSession(chatID) {
+    this[chatID] = { spentCities: new Set() };
+  },
+  deleteSession(chatID) {
+    delete this[chatID];
+  }
+};
+
+function randomCity(arrCities) {
+  return arrCities[helpers.getRandomNumber(arrCities.length)];
+}
+
+function lastValidLetter(str) {
+  let lastLetter = str[str.length - 1];
+  let invalidLetters = ["ь", "ъ"];
+  let i = 2;
+
+  while (invalidLetters.includes(lastLetter)) {
+    lastLetter = str[str.length - i];
+    i++;
+  }
+  return lastLetter;
+}
+
+function validateMessage(message) {
+  let invalidSymbols = [" ", ".", ","];
+  if (
+    message.length <= 3 ||
+    message.length > 30 ||
+    invalidSymbols.findIndex(item => message.includes(item)) > -1
+  ) {
+    return "Название города должно содержать одно слово, \nбез пробелов и знаков препинания, \nот 3 до 30 символов.";
+  }
+  return null;
+}
+
+function checkCityInDB(chatID, city, letter) {
+  if (letter != city[0]) {
+    return "Нужно назвать город на букву " + letter.toUpperCase();
+  }
+  if (sessions[chatID].spentCities.has(city)) {
+    return "Этот город уже был назван!";
+  }
+  return null;
+}
+
+function selectCityByLetter(chatID, letter, cities) {
+  let findCities = cities.filter(
+    item => item[0] == letter && !sessions[chatID].spentCities.has(item)
+  );
+  if (findCities.length === 0) {
+    return null;
+  } else return randomCity(findCities);
```

Comment:
always use brackets with else
```suggestion
  } else {
    return randomCity(findCities);
  }
```